### PR TITLE
[INJIMOB-1578] add api for key management and data signing

### PR DESCRIPTION
**Pull request by abhip2565**

### CODE REVIEW

File: Sources/securekeystore/Biometrics/BiometricsImpl.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import Foundation
+
+class BiometricsImpl: BiometricsProtocol {
+    
+    
+    private static var popUpTitle: String = ""
+    private static var popupDescription: String = ""
+
+    
+    private let requiredDelay: TimeInterval = 15
```
Comment (swatigoel) [inline Sources/securekeystore/Biometrics/BiometricsImpl.swift:11]: rename it to `authOffsetInSeconds`

File: Sources/securekeystore/KeychainManager/KeychainManagerImpl.swift [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        
+        var result: AnyObject?
+        let status = SecItemCopyMatching(query as NSDictionary, &result)
+        
+        guard status == errSecSuccess else {
+            print("Failed to retrieve keys from keychain: \(status)")
+            return
+        }
+        
+        print("key dumpppp",result ?? [])
```
Comment (swatigoel) [inline Sources/securekeystore/KeychainManager/KeychainManagerImpl.swift:56]: why do we need this log?

File: Sources/securekeystore/SeureKeystoreImpl.swift [hunk 3]
```diff
@@ -0,0 +1,237 @@
+import Foundation
+import Security
+
+@available(iOS 13.0, *)
+public class SecureKeystoreImpl:SecureKeystoreProtocol {
```
Comment (swatigoel) [inline Sources/securekeystore/SeureKeystoreImpl.swift:5]: typo mistake in filename
filename should `SecureKeystoreImpl.swift`

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: Sources/securekeystore/Biometrics/BiometricsImpl.swift (hunk 1 above)
Comment (abhip2565) [reply Sources/securekeystore/Biometrics/BiometricsImpl.swift:11]: Done

File: Sources/securekeystore/KeychainManager/KeychainManagerImpl.swift (hunk 2 above)
Comment (abhip2565) [reply Sources/securekeystore/KeychainManager/KeychainManagerImpl.swift:56]: Not needed, added for debugging purposeless. Removing 😅

File: Sources/securekeystore/SeureKeystoreImpl.swift (hunk 3 above)
Comment (abhip2565) [reply Sources/securekeystore/SeureKeystoreImpl.swift:5]: yes right

**Review by swatigoel — APPROVED** [review]

**Pull request merged**